src/signals/settings/users/containers/Detail/components/UserForm/index.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { themeSpacing } from '@datapunt/asc-ui';
+import styled from 'styled-components';
+
+import RadioButtonList from 'signals/incident-management/components/RadioButtonList';
+
+import Input from 'components/Input';
+import Label from 'components/Label';
+import FormFooter from 'components/FormFooter';
+
+const Form = styled.form`
+  width: 100%;
+`;
+
+const FieldGroup = styled.div`
+  & + & {
+    margin-top: ${themeSpacing(8)};
+  }
+`;
+
+const StyledFormFooter = styled(FormFooter)`
+  position: fixed;
+`;
+
+const statusOptions = [
+  { key: 'true', value: 'Actief' },
+  { key: 'false', value: 'Niet actief' },
+];
+
+const UserForm = ({ data, onCancel, onSubmitForm }) => (
+  <Form action="" data-testid="detailUserForm">
+    <FieldGroup>
+      <Input
+        defaultValue={data.username}
+        id="username"
+        name="username"
+        label="E-mailadres"
+        readOnly
+        disabled
+      />
+    </FieldGroup>
+
+    <FieldGroup>
+      <Input
+        defaultValue={data.first_name}
+        id="first_name"
+        name="first_name"
+        label="Voornaam"
+      />
+    </FieldGroup>
+
+    <FieldGroup>
+      <Input
+        defaultValue={data.last_name}
+        id="last_name"
+        name="last_name"
+        label="Achternaam"
+      />
+    </FieldGroup>
+
+    <FieldGroup>
+      <Label as="span">Status</Label>
+      <RadioButtonList
+        defaultValue={`${data.is_active}`}
```

Comment:
I think a `data.is_active.toString()` is more explicit.

Reply:
It is, but that will throw an error when `data` is an empty object